package.json

```diff
@@ -15,7 +15,8 @@
     "check-format": "yarn prettier --list-different",
     "test": "jest --verbose",
     "dev": "ts-node-dev --respawn --transpileOnly ./src/samples/agency.ts",
-    "prod": "rm -rf build && yarn compile && DEBUG=aries-framework-javascript node ./build/samples/agency.js",
+    "prod:start": "DEBUG=aries-framework-javascript node ./build/samples/agency.js",
+    "prod": "rm -rf build && yarn compile && yarn prod:start",
```

Comment:
What about: 
```
    "prod:start": "node ./build/samples/agency.js",
    "prod:debug": "DEBUG=aries-framework-javascript && yarn prod:start",
    "prod:build": "rm -rf build && yarn compile",
```

Reply:
I was a little bit confused because both commands are starting the app.

Reply:
Great suggestion. Will change 👍